### ISSUE TITLE
Unable to change output format in 1.14.3

### ISSUE THREAD

**Issue by mozoh** [danielwelch/hassio-zigbee2mqtt#371]
I'm trying to configure zigbee2mqtt with the following config field:
```
experimental:
  output: 'attribute_and_json'
```
but seems like it does not work, and i've got the message in the supervisor logs:
```
20-04-18 01:31:57 WARNING (MainThread) [supervisor.addons.validate] Unknown options experimental
```
I am using HassOS 4.12 and addon version 1.14.3

**Comment by ciotlosm** [comment]
@mozoh can you please post your add-on config?

**Comment by mozoh** [comment]
```
data_path: /config/zigbee2mqtt
devices: devices.yaml
groups: groups.yaml
homeassistant: true
permit_join: false
mqtt:
  base_topic: zigbee2mqtt
  server: 'mqtt://core-mosquitto'
  user: mqtt
  password: my mqtt password
serial:
  port: /dev/ttyACM1
advanced:
  pan_id: 6754
  channel: 11
  network_key: my network key
  availability_blacklist: []
ban: []
whitelist: []
queue: {}
socat:
  enabled: false
  master: 'pty,raw,echo=0,link=/dev/ttyZ2M,mode=777'
  slave: 'tcp-listen:8485,keepalive,nodelay,reuseaddr,keepidle=1,keepintvl=1,keepcnt=5'
  restartdelay: 1
  initialdelay: 1
  options: '-d -d'
  log: false
experimental:
  output: attribute_and_json
```

**Comment by ciotlosm** [comment]
According to (docs)[https://www.zigbee2mqtt.io/information/configuration.html] you should have:
```
experimental:
  output: 'json'
```

**Comment by mozoh** [comment]
Please read the docs more careful:
```
# Optional: experimental options
experimental:
  # Optional: MQTT output type: json, attribute or attribute_and_json (default: shown below)
  # Examples when 'state' of a device is published
  # json: topic: 'zigbee2mqtt/my_bulb' payload '{"state": "ON"}'
  # attribute: topic 'zigbee2mqtt/my_bulb/state' payload 'ON"
  # attribute_and_json: both json and attribute (see above)
  output: 'json'
```
I need to use exactly `attribute_and_json`

**Comment by ciotlosm** [comment]
You need to use ' around your value like:
```
experimental:
  output: 'attribute_and_json'
```

Also keep in mind that this repository is for zigbee2mqtt home assistant addon. The settings discussed and considering you're changing the actual `yaml` hint that you should be adressing such issues in http://github.com/koenkk/zigbee2mqtt

**Comment by mozoh** [comment]
But hassio automatically removes '' on config save.

Btw, the issue is:
```
20-04-18 01:31:57 WARNING (MainThread) [supervisor.addons.validate] Unknown options experimental
```
and not the even `output` field

**Comment by ciotlosm** [comment]
Ok, I tested this out and it seems the problem is missing support for `output` inside `experimental`. See: https://github.com/danielwelch/hassio-zigbee2mqtt/blob/master/zigbee2mqtt/config.json#L146

There must have been something wrong in your config because the error should have been:
`20-09-17 13:34:04 WARNING (MainThread) [supervisor.addons.validate] Unknown options output`

This requires a PR to fix.

**Comment by ciotlosm** [comment]
I've added https://github.com/danielwelch/hassio-zigbee2mqtt/pull/389 to support this.

**Comment by danielwelch** [comment]
closed by #389, please re-open if still having issues.
